frontend/kesaseteli/employer/src/__tests__/utils/components/render-page.tsx

```diff
@@ -7,20 +7,21 @@ import { NextRouter } from 'next/router';
 import React from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import getDefaultReactQueryTestClient from 'shared/__tests__/utils/react-query/get-default-react-query-test-client';
-import { render, RenderResult } from 'shared/__tests__/utils/test-utils';
+import { act,render } from 'shared/__tests__/utils/test-utils';
```

Comment:
😃 
```suggestion
import { act, render } from 'shared/__tests__/utils/test-utils';
```

Reply:
For some reason prettier does not fix these